src/daft-csv/test/iris_tiny_double_quote_escape.csv

```diff
@@ -0,0 +1,22 @@
+"""sepal.""length",sepal.width,petal.length,petal.width,variety
+5.1,3.5,1.4,.2,"Setosa"
```

Comment:
Let's add some actually useful escaped test-cases? E.g. `"Setosa,Sentosa"` :)

Reply:
+1, at a minimum it would be a good idea to add double quotes to some of the variety values (e.g. ""Setosa"") in the below rows, to ensure that the double quote escape flag works as expected for rows beyond just the header.